src/localization/locales.js

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2020 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+export const locales = {
+    en : 'English',
+    fi : 'Suomi',
+    pt: 'Português'
+}
+
+export const emptyLocales = {
```

Comment:
you can generate this value from __locales__ by doing the following:
```
export const emptyLocales = Object.keys(locales)
  .reduce((acc, key) => ({...acc, [key]: null}), {});
```

Reply:
To get the same values we should use keyMirror, so we can combine your solution with this:
```suggestion
export const LOCALES = keyMirror(Object.keys(locales) .reduce((acc, key) => ({...acc, [key]: null}), {}) );
```
Just need to import keyMirror to this file and update the places that will import this LOCALES const. I think now it is better to change the const name from **emptyLocales** to just **LOCALES** as keyMirror will return a object like this:

`{
en: "en",
fi: "fi",
pt: "pt"
}`

Reply:
you don't need to use keyMirror in this scenario, you can simply do the following:
```
export const emptyLocales = Object.keys(locales)
  .reduce((acc, key) => ({...acc, [key]: key}), {});
```

Reply:
Thanks that will work